TYPO3.Flow/Classes/TYPO3/Flow/Persistence/Doctrine/Mapping/Driver/FlowAnnotationDriver.php

```diff
@@ -732,7 +737,13 @@ protected function evaluatePropertyAnnotations(ClassMetadataInfo $metadata)
                             break;
                         default:
                             if (strpos($propertyMetaData['type'], '\\') !== false) {
-                                if ($this->reflectionService->isClassAnnotatedWith($propertyMetaData['type'], Flow\ValueObject::class)) {
+                                if ($valueObjectAnnotation = $this->reflectionService->getClassAnnotation($propertyMetaData['type'], Flow\ValueObject::class)) {
```

Comment:
Yeah ok, now I get it too - the "2" is necessary to also skip out of the switch statement - it basically functions as a "break & continue;"